cluster/aws/util.sh

```diff
@@ -352,6 +352,34 @@ function wait-for-instance-running {
   done
 }
 
+function allocate-elastic-ip {
+  $AWS_CMD allocate-address --domain vpc --output text | cut -f3
```

Comment:
Is https://google-styleguide.googlecode.com/svn/trunk/shell.xml the proper styleguide to adhere or is it some other?